database/dataGenerator.js

```diff
@@ -0,0 +1,65 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const faker = require('faker');
+// const db = require('./index.js');
+
+
+const dateGenerator = () => {
+  const year = Math.floor(Math.random() * 2) + 2017;
+  let month = Math.floor(Math.random() * 11) + 1;
+  let day = Math.floor(Math.random() * 27) + 1;
+  if (month < 10) {
+    month = `0${month}`;
+  }
+  if (day < 10) {
+    day = `0${day}`;
+  }
+  return `${year}-${month}-${day}`;
+};
+
+const stringPicker = () => {
```

Comment:
you may also consider using faker's 'Random' methods to help you with picking a value from an array
https://github.com/Marak/faker.js/wiki/Random 
faker.random.arrayElement(array[])